### ISSUE TITLE
Instant revamp bake-off

### ISSUE THREAD

**Issue by mintcloud** [0xProject/0x-instant#1]
[0x Instant](https://blog.0xproject.com/introducing-0x-instant-7314c786d743?source=search_post---------0) is a configurable token swap widget that connects to 0x API liquidity. Instant code is in the form of a [package](https://github.com/0xProject/0x-monorepo/tree/development/packages/instant) in the 0x-monorepo repository.
Instant is not currently being maintained by 0x Labs, but it has proven to attract a fair amount of developers experimenting with 0x for the first time. While there haven’t been any significant live applications of the widget since its launch (probably the only one live now is on ZRX Portal, deployed by 0x Labs), there are still a lot of questions on the dedicated [Discord channel](https://discord.gg/n82t7Mm), which signals there is an appetite by developers to deploy hosted applications.

0x Labs proposes to fund the development of a revamped version of Instant, addressing the most commonly requested features listed in the minimum requirements.

**Minimum requirements**

* Make Instant use swap/v1/quote 0x API endpoints (specs WIP, you can refer to swap/v0/quote in the official docs [1](https://0x.org/docs/api#get-swapv0quote) [2](https://0x.org/docs/guides/swap-tokens-with-0x-api)) instead of AssetSwapper.
    * support ERC20<>ERC20
    * support ETH unwrapping natively (sell ERC20s for ETH)
* Utilize affiliate fees via swap/v1 ([ref PR](https://github.com/0xProject/0x-api/pull/310)) instead of forwarder contract.
* Create a comprehensive guide (better if with video walkthroughs and/or screenshots) for developers to customize token pairs and other settings.
* Publish guide at [0x.org/docs](http://0x.org/docs) .
* Host a live version of Instant.

**Nice to have**

* Design revamp
* Revamp and host Instant Configurator to make it easier to add custom tokens

**Logistics of the bounty**

0x Labs will open up a gitcoin bounty (link WIP), and will give applicants **2 weeks to propose their development plan directly in the gitcoin page. The deadline to submit the plan is Sunday, October 3rd**. It will be asked to applicants to present their plans at the upcoming Ecosystem and Governance call (scheduled for [Wednesday, October 7th, 21pm UTC](https://github.com/0xProject/pm/issues/32)). 

At the end of this period, 0x Labs will assign the bounty to the developer who prepared the best proposal. Developer experience (via github contributions and similar) will be assessed, and it will be given priority to who expresses interest in continued maintenance of the product.

The bounty will be sent to the developer once the planned work is finished and verified by 0x Labs. In the meantime, 0x Labs will proceed at moving 0x Instant package into its own independent repository, so to be ready once the project is assigned.

To summarize:

- STEP 1. Interested developers read the issue, get familiar with the package and minimum requirements
- STEP 2. 0x Labs opens a gitcoin bounty
- STEP 3. Developer applies to bounty by proposing a plan directly in the gitcoin page. The more details in the plan the better - we don't want just applicants saying "I can work on this project". Prior experience on 0x stack will be appreciated but not required. Deadline to share the plan is Sunday, October 3rd
- STEP 4. If further clarification is needed, applicants will be invited at the next Ecosystem call (https://github.com/0xProject/pm/issues/32) to present the plan
- STEP 5. The bounty is assigned to the best proposal
- STEP 6. The developer works on the project
- STEP 7. 0x Labs review, the developer is paid

**Comment by mintcloud** [comment]
Issue closed by #2

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Started** 3. Submitted 4. Done 

<hr>

__Work for 10000.0 ZRX (7991.66 USD @ $0.8/ZRX) has been submitted by__:

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/0xproject/0x-instant/1/100023724)
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/0xproject/0x-instant/1/100023724).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://discord.gg/gitcoin/'>Gitcoin's Discord</a>
* $5,521,290.04 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)
